conf/test.config

```diff
@@ -10,17 +10,54 @@
 params {
   config_profile_name = 'Test profile'
   config_profile_description = 'Minimal test dataset to check pipeline function'
+  
   // Limit resources so that this can run on GitHub Actions
   max_cpus = 2
-  max_memory = 6.GB
+  max_memory = 4.GB
   max_time = 48.h
 
   // Input data
-  // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-  // TODO nf-core: Give any required params for the test so that command line flags are not needed
   single_end = false
-  readPaths = [
-    ['Testdata', ['https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R1.tiny.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R2.tiny.fastq.gz']],
-    ['SRR389222', ['https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub1.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub2.fastq.gz']]
+
+  // run processes
+
+  //run_salmon_selective_alignment = true
+  //run_star = true
+  //run_salmon_alignment_based_mode = true
+  //run_htseq_uniquely_mapped = true
+  //mapping_statistics = true
+
+
```

Comment:
leftover comments?

Reply:
Yeah - we were still deciding which processes to run in the test.conf file. Have tidied this up now